replica/database.hh

```diff
@@ -1434,6 +1434,8 @@ private:
     friend db::data_listeners;
     std::unique_ptr<db::data_listeners> _data_listeners;
     std::unique_ptr<db::toppartitions_data_listener> _tp_listener;
+    timer<seastar::lowres_clock> _tp_timer;
```

Comment:
Pleas avoid using abbreviations in the database class.

Reply:
Fixed.